### PR TITLE
OPS-1017: Adding output if chimp job group doens't exist

### PR DESCRIPTION
**Pull request by gregcoit**

### CODE REVIEW

File: lib/right_chimp/Chimp.rb [hunk 1]
```diff
@@ -1138,6 +1138,7 @@ def chimpd_wait_until_done
               all = ChimpDaemonClient.retrieve_group_info(@chimpd_host, @chimpd_port, @group, :all)
             rescue RestClient::ResourceNotFound
               sleep 5
+              $stdout.print "\nStill waiting for group #{@group}..."
```
Comment (markitoxs) [inline lib/right_chimp/Chimp.rb]: Can we actually change this to something along the lines of: WARN: Group "#{@group}" does not exist ? I tihnk that is a little bit more specific on what the issue really is.

**Comment by gregcoit** [comment]
Yes we can!  :)  Updated

**Comment by markitoxs** [comment]
Please update the version number too.

**Comment by markitoxs** [comment]
Please merge!

**Pull request merged**